src/components/atomic-swap/ProposalBalanceTable.js

```diff
@@ -15,12 +15,17 @@ import { t } from "ttag";
  * @param {DisplayBalance[]} balance
  */
 export function ProposalBalanceTable ({ partialTx, wallet, balance }) {
-    const sendingBalances = balance.filter(b => b.sending > 0);
     const receivingBalances = balance.filter(b => b.receiving > 0);
+    // If the wallet participates in the proposal with a token with zero balance, it is displayed as "sending"
+    const sendingBalances = balance.filter(b => {
+        return b.sending > 0 || b.sending === -0;
```

Comment:
-0? And why not `>=`?

Reply:
This was a leftover from a previous implementation: simplified on [45499cf](https://github.com/HathorNetwork/hathor-wallet/pull/402/commits/45499cf8b842e173854c96db983188bcec317bd1).